pkgs/by-name/sc/scom/package.nix

```
@@ -0,0 +1,33 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+}:
+stdenv.mkDerivation rec {
+  pname = "scom";
+  version = "1.0";
+
+  src = fetchFromGitHub {
+    owner = "crash-systems";
+    repo = pname;
```

```suggestion
    repo = "scom";
```
Although it feel natural to link repository name to the package name, this is actually an anti-pattern, and can create some unwanted effects:
- #277994